app/data/ukis_main.json

```diff
@@ -0,0 +1,5112 @@
+                },
+                {
+                    "type": "interstitial",
+                    "id": "innovation-activities-block-26",
+                    "sections": [
+                        {
+                            "description": "<p> </p><p>The next set of questions covers innovation relating to <em>goods and services</em>.</p><p> </p><p>You will first be asked to provide information about any new or significantly improved goods introduced by your business; and second to provide information relating to any new or significantly improved services.</p><p> </p><p>You will also be asked to provide estimates for turnover in the calendar year 2016 relating to new or improved goods or services introduced during the period 1 January 2014 to 31 December 2016.</p>",
+                            "number": "2",
+                            "questions": [
+                                {
+                                    "description": "",
+                                    "answers": [],
+                                    "type": "General",
+                                    "id": "innovation-activities-question-26",
+                                    "title": ""
+                                }
+                            ],
+                            "id": "innovation-activities-section-26",
+                            "title": "Innovation Activities"
+                        }
+                    ]
+                }
+            ],
+            "id": "innovation-activities-section-26-group-26",
+            "title": "Innovation Activities"
+        },
+        {
+            "blocks": [
+                {
+                    "type": "questionnaire",
+                    "id": "innovation-activities-block-27",
+                    "sections": [
+                        {
+                            "description": "<p>3 year period 1 January 2014 to 31 December 2016.</p>",
+                            "number": "2",
+                            "questions": [
+                                {
+                                    "description": "",
+                                    "number": "2.19",
+                                    "guidance": [
+                                        {
+                                            "description": "",
+                                            "list": [
+                                                "all new or significantly improved goods (for example, improvement in quality or distinct user benefits)",
+                                                "goods innovations that are new to the business, even if they are not new to the market",
+                                                "all product innovations, regardless of their origin"
+                                            ],
+                                            "title": "Include:"
+                                        },
+                                        {
+             
```

Comment:
Guidance appears in a different place than in the slides, Jenny has updated the slides so just needs to be regenerated